### ISSUE TITLE
Preventing Default

### ISSUE THREAD

**Issue by gildir** [forumone/jquery.selectability.js#1]
Trying to call preventDefault() on a select does not prevent selectability from showing the selection. Example: I have a select that opens a popup window, when the user closes the popup I want the select to be back at "select a X:" or whatever and not "the last thing you chose". $('select').val("foo") won't work until the next change event fires, but opening the popup happens in the change handler, so $('select').trigger('change') is not an option either.

Basically, can it check if preventDefault() has been called and actually prevent the default action?
